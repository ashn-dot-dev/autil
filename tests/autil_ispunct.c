#define AUTIL_IMPLEMENTATION
#include "../autil.h"
#include "test.h"

int
main(void)
{
    for (int i = 0; i < 256; ++i) {
        printf("%c", autil_ispunct(i) ? '1' : '0');
    }
    fputc('\n', stdout);
    return EXIT_SUCCESS;
}
