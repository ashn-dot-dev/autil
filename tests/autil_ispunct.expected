0000000000000000000000000000000001111111111111110000000000111111100000000000000000000000000111111000000000000000000000000001111000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
